src/internals/strings/Strings.ts

```diff
@@ -21,57 +22,512 @@ export namespace Strings {
+   */
+  export interface Replace<from extends string, to extends string> extends Fn {
+    output: Impl.Replace<this["args"][0], from, to>;
+  }
+
+  /**
+   * Cut a slice of a string out from a start index to an end index.
+   * @param args[0] - The string to slice.
+   * @param start - The start index.
+   * @param end - The end index.
+   * @returns The sliced string.
+   * @example
+   * ```ts
+   * type T0 = Call<Strings.Slice<1,9>,"1234567890">; // "23456789"
+   */
+  export interface Slice<start extends number, end extends number> extends Fn {
+    output: Pipe<
+      Impl.StringToTuple<this["args"][0]>,
+      [Tuples.Take<end>, Tuples.Drop<start>, Join<"">]
+    >;
+  }
+
+  /**
+   * Split a string into a tuple of strings.
+   * @param args[0] - The string to split.
+   * @param sep - The separator to split the string with.
+   * @returns The split string.
+   * @example
+   * ```ts
+   * type T0 = Call<Strings.Split<",">,"a,b,c">; // ["a","b","c"]
+   * ```
+   */
   export interface Split<sep extends string> extends Fn {
-    output: H.Split<this["args"][0], sep>;
+    output: Impl.Split<this["args"][0], sep>;
   }
 
+  /**
+   * Repeat a string a number of times.
+   * @param args[0] - The string to repeat.
+   * @param times - The number of times to repeat the string.
+   * @returns The repeated string.
+   * @example
+   * ```ts
+   * type T0 = Call<Strings.Repeat<3>,"Hello! ">; // "Hello! Hello! Hello! "
+   * ```
+   */
+  export interface Repeat<times extends number> extends Fn {
+    output: Impl.Repeat<this["args"][0], H.Iterator.Iterator<times>>;
+  }
+
+  /**
+   * Check if a string starts with a substring.
+   * @param args[0] - The string to check.
+   * @param str - The substring to check for.
+   * @returns Whether the string starts with the substring.
+   * @example
+   * ```ts
+   * type T0 = Call<Strings.StartsWith<"abc">,"abcdef">; // true
+   * type T1 = Call<Strings.StartsWith<"abc">,"defabc">; // false
+   * ```
+   */
+  export interface StartsWith<str extends string> extends Fn {
+    output: this["args"][0] extends `${str}${string}` ? true : false;
+  }
+
+  /**
+   * Check if a string ends with a substring.
+   * @param args[0] - The string to check.
+   * @param str - The substring to check for.
+   * @returns Whether the string ends with the substring.
+   * @example
+   * ```ts
+   * type T0 = Call<Strings.EndsWith<"abc">,"abcdef">; // false
+   * type T1 = Call<Strings.EndsWith<"abc">,"defabc">; // true
+   * ```
+   */
+  export interface EndsWith<str extends string> extends Fn {
+    output: this["args"][0] extends `${string}${str}` ? true : false;
+  }
+
+  /**
+   * Split a string into a tuple of each character.
+   * @param args[0] - The string to split.
+   * @returns The splited string.
+   * @example
+   * ```ts
+   * type T0 = Call<Strings.ToTuple,"abc">; // ["a","b","c"]
+   */
+  export interface ToTuple extends Fn {
+    output: Impl.StringToTuple<this["args"][0]>;
+  }
```

Comment:
Do we need this given that we have `Strings.Split`? there are many ways we could turn strings into tuples, so this function is looks confusing to me